uk_bin_collection/uk_bin_collection/councils/StratfordUponAvonCouncil.py

```
@@ -1,94 +1,125 @@
+            date_str = date_cell.text.strip()
+            try:
+                date_obj = datetime.strptime(date_str, "%A, %d/%m/%Y")
+            except ValueError:
+                continue
+
+            collection_info = [
+                cell.get("title", "Not Collected")
+                for cell in row.select("td.text-center")
+            ]
+
+            for bin_type, status in zip(column_headers, collection_info):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)